src/content-tags-drawer/data/api.js

```diff
@@ -29,7 +29,12 @@ export const getTaxonomyTagsApiUrl = (taxonomyId, options = {}) => {
   return url.href;
 };
 export const getContentTaxonomyTagsApiUrl = (contentId) => new URL(`api/content_tagging/v1/object_tags/${contentId}/`, getApiBaseUrl()).href;
-export const getContentDataApiUrl = (contentId) => new URL(`/xblock/outline/${contentId}`, getApiBaseUrl()).href;
+export const getContentDataApiUrl = (contentId) => (
+  // Check if the contentId belongs to a V2 library component
+  contentId.startsWith('lb:')
+    ? new URL(`/api/libraries/v2/blocks/${contentId}/`, getApiBaseUrl()).href
+    : new URL(`/xblock/outline/${contentId}`, getApiBaseUrl()).href
+);
```

Comment:
This is not a huge deal, but I feel it might be better to keep the URL function "dumb" and keep this if elsewhere. 

i.e. have two function `getXBlockContentDataApiURL` and `getLibraryContentDataApiUrl` and the decision of which to be used can be elsewhere. 

Does that make sense?

Reply:
Good point, yup I agree it would be cleaner to remove the logic from the URL function and keep it simple. I've updated it with your suggestion and put the logic where the URL functions are called in `getContentData`.